### PR TITLE
CELLMIA reader: fix coordinate mapping

### PR DESCRIPTION
**Pull request by simleo**
Applies two fixes to the CELLMIA reader:

1. Do not use the config to get field info, since this is fixed and known in advance
2. Map `cell row` to the y coord and `cell col` to the x coord. This is consistent with the array vs image nomenclature and has been verified in the [overlay notebook](https://github.com/CellMigStandOrg/CMSO-Samples/blob/2b69fe5ebc29f8c4a4150f26656f349e18567780/overlay.ipynb)

### CODE REVIEW

File: biotracks/readfile.py [hunk 1]
```diff
@@ -377,21 +377,20 @@ class CellmiaReader(AbstractReader):
...
-        # subtract 1 for consistency across software
-        df[cellmia_link_id] -= 1
+                         usecols=self.CELLMIA_COLS)
         df.reset_index(inplace=True)
-        df.columns = [cmso.OBJECT_ID, cmso.LINK_ID, cmso.FRAME_ID,
-                      cmso.X_COORD, cmso.Y_COORD]
+        cols = [cmso.OBJECT_ID] + self.CMSO_COLS
+        df.columns = cols
+        cols[-1], cols[-2] = cols[-2], cols[-1]
+        df = df[cols]
```
Comment (gsergeant) [inline biotracks/readfile.py:390]: Could you explain what is happening in this last line? Why is this needed?

Comment (simleo) [reply biotracks/readfile.py:390]: It's the last two lines actually. `df.columns = cols` maps `cell row` to `cmso.Y_COORD` and `cell col` to `cmso.X_COORD`, but it does not change column order. The dataframe would still be correct, but it would write out `cmso_y_coord` before `cmso_x_coord`. It's just to keep things tidy.

Comment (gsergeant) [reply biotracks/readfile.py:390]: I meant line 390, is this just kind of resetting the dataframe?

Comment (simleo) [reply biotracks/readfile.py:390]: No, it's just reordering columns.

Comment (gsergeant) [reply biotracks/readfile.py:390]: I see now, thanks. 👍
